### DISCUSSION TITLE
Snippets in any filetypes

### DISCUSSION THREAD

**Discussion by korchizhinskiy** [hrsh7th/nvim-cmp#1305] [Q&A]
How i can remove snippets in any filetypes. For example, in Python (.py) files.
[#759](https://github.com/hrsh7th/nvim-cmp/discussions/759#discussioncomment-2052826) i tried, but this remove snippets in all of files, that interferes completing in html and css.

What i may doing for switch off snippets in any filetypes?
![image](https://user-images.githubusercontent.com/100942887/201685619-9e03098b-3339-47f3-9fe1-1cd3bfc944ad.png)

**Comment by ehaynes99** [comment]
https://github.com/hrsh7th/nvim-cmp/blob/8bbaeda725d5db6e4e1be2867a64b43bf547cf06/doc/cmp.txt#L848-L859

However, `nvim-cmp` doesn't include snippets on its own. You had to configure them yourself, and it looks like you just have some source configured that got way out of hand...
